docs/installation.md

```diff
@@ -16,10 +16,29 @@ $ npm install kinto --save
 
 Dev and production ready assets are available in the [`gh-pages` branch of the repository](https://github.com/Kinto/kinto.js/tree/gh-pages). In general, you should download these files and ship them along your own projects, though you can also link them during development:
 
-- Dev version, including source maps: [kinto-1.1.2.js](http://npmcdn.com/kinto@1.1.2/dist/kinto-1.1.2.js)
-- Production version, minified, no source maps: [kinto-1.1.2.min.js](http://npmcdn.com/kinto@1.1.2/dist/kinto-1.1.2.min.js)
-- Minimalist version, without polyfills [kinto-1.1.2.noshim.js](http://npmcdn.com/kinto@1.1.2/dist/kinto-1.1.2.noshim.js)
+- Dev version, including source maps: [kinto-1.2.0.js](http://npmcdn.com/kinto@1.2.0/dist/kinto-1.2.0.js)
+- Production version, minified, no source maps: [kinto-1.2.0.min.js](http://npmcdn.com/kinto@1.2.0/dist/kinto-1.2.0.min.js)
+- Minimalist version, without polyfills [kinto-1.2.0.noshim.js](http://npmcdn.com/kinto@1.2.0/dist/kinto-1.2.0.noshim.js)
 
 > #### Notes
 >
 > Only stable tags are released as dist files; if you plan on using latest versions from master, you need to [build them manually](hacking.md#generating-dist-files).
+
+
+### Subresource integrity
+
+To make sure that you are using the right code when loading from a CDN, you can use subresource
+integrity with the hash provided below:
```

Comment:
nit: add a link to https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity ?